### PR TITLE
updated to Python 3.8 and the latest s2i-thoth version

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

## Related Issues and Dependencies
n/a

## This introduces a breaking change

- [ ] Yes
- [X] No

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ pytest-cov = "*"
 pylint = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.8"
```
Comment (fridex) [inline Pipfile:16]: Is there a need to relock `Pipfile.lock`? Or do deps stay the same (I haven't tried it).

**Comment by saisankargochhayat** [comment]
Looks good! Other components that use this would need py38 image in deployment?

**Comment by goern** [comment]
> Looks good! Other components that use this would need py38 image in deployment?

ja, lets slowly migrate them all :)

**Comment by harshad16** [comment]
Done by #321

**Pull request closed without merging**